docs/src/whatsnew/3.1.rst

```diff
@@ -93,6 +93,53 @@ This document explains the changes made to Iris for this release
 #. `@trexfeathers`_ fixed the "anomaly_log_colouring" gallery example to be compatible with the latest Matplotlib usage
    (:pull:`4115`)
 
+===================================
+Bugs fixed in v3.0.4 (22 July 2021)
+===================================
```

Comment:
Should these not be in the 3.0.rst whats new not the 3.1.rst?

Reply:
Oh dear.
I thought not, but now I look back at what we did at 2.2.1 / 2.3.0 I think I was wrong.
So yes, I suppose the bug fix sections should all go into an updated version of the 3.0 whatsnew.

But I find it a  bit odd that the release notes do **_not_** explain everything that changed since the previous release
( you have to read the _updated_ version of the previous release notes, too )
Does this seem right to you @lbdreyer ?

Reply:
When I look at the v3.0 what's new I expect to see the changes for all v3.0 releases. This includes 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4.
If after having cut 3.1, we have to go back and cut another 3.0 patch release (v3.0.5), I would expect to see these changes as part of the 3.0 what's new.